LDM-592.tex

```diff
@@ -1126,6 +1126,14 @@ \section{Science Quality and Reliability Engineering}
 
 }
 
+\usecase{SQR17}{Retrieve Job data based on metric values.}{%
+
+\item
+The continuous integration system publishes \texttt{Job} data including \texttt{Metric}s, \texttt{Measurement}s, and data associated with the \texttt{Measurement}s for display within SQuaSH.
+At some later point, the SQuaRE scientist retrieves all \texttt{Job}s that were run in the last month where the measurement of the photometric repeatablity is greater than 15 mas.
```

Comment:
This is a distinct part of the use case so should have its own `\item`.

Reply:
OK, I'll try again.